### ISSUE TITLE
Wrong theming for mermaid

### ISSUE THREAD

**Issue by Commandcracker** [squidfunk/mkdocs-material#5300]
### Bug description

- [Grouping / Box ](https://mermaid.js.org/syntax/sequenceDiagram.html#grouping-box) lables are hard to see, because they dont change color on theme change.
- [Notes](https://mermaid.js.org/syntax/sequenceDiagram.html#notes) dont fit the theme.

- > [Messages](https://mermaid.js.org/syntax/sequenceDiagram.html#messages) and arrows are hard to see on light theme if they are on top of a box with an inappropriate color.
  > There might be nothing that can be done here.

### Related links

Links are provided in the description.

### Reproduction

[minimal reproduction.zip](https://github.com/squidfunk/mkdocs-material/files/11130228/minimal.reproduction.zip)

### Steps to reproduce

1. Run the development server
2. See the wrong theming

### Browser

Firefox

### Before submitting

- [X] I have read and followed the [bug reporting guidelines](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/).
- [X] I have attached links to [the documentation](https://squidfunk.github.io/mkdocs-material), and possibly related [issues](https://github.com/squidfunk/mkdocs-material/issues) and [discussions](https://github.com/squidfunk/mkdocs-material/discussions).
- [X] I assure that I have [removed all customizations](https://squidfunk.github.io/mkdocs-material/contributing/reporting-a-bug/#remove-customizations) before submitting this bug report.
- [X] I have attached a __.zip file__ with a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Comment by Commandcracker** [comment]
### Visuals

![grafik](https://user-images.githubusercontent.com/49335821/229160071-786fb948-cb7e-4e80-b006-0c5cf8b6b829.png)
![grafik](https://user-images.githubusercontent.com/49335821/229160140-ddaaf791-4f7c-4373-9959-9007e38e4f36.png)
```mermaid
sequenceDiagram
    autonumber
    actor Client
    box rgb(33,66,99) Server
        participant Sanic
        participant YT-DLP
        participant Cache
        participant FFmpeg
        participant Sanjuuni
    end
    Note over Cache: Folder on server
    Note over Sanic: Web server
    Client-)Sanic: Sends a media request
    Note over Sanic: If the media is from spotify<br/>use spotipy to get information to<br/>create a serch term
    Sanic--)YT-DLP: Get information about the media
    Sanic-)Client: Tells the client<br/>inforamtion about the media
    Sanic--)YT-DLP: Downloads media
    YT-DLP-)Cache: Stores media in cache
    Sanic--)FFmpeg: Convert audio to dfpwm
    FFmpeg-)Cache: Stores audio in cache
    Sanic--)Sanjuuni: Convert video to 32vid
    Sanjuuni-)Cache: Stores video in cache
    Cache->Cache: Deletes original downloaded media
    loop Streaming
        Client-)Sanic: Request video and audio chunks
        Cache-)Sanic: Reads video and audio chunks
        Sanic-)Client: Sends video and audio chunks
        Note over Client: Receives and displays the media
    end
```

**Comment by squidfunk** [comment]
Thanks for reporting! Fixed in 97fd112fe.

<img width="775" alt="Bildschirm­foto 2023-04-07 um 13 26 39" src="https://user-images.githubusercontent.com/932156/230601276-b53772d8-1442-4d1b-b9e9-d578aa9d37d8.png">

<img width="768" alt="Bildschirm­foto 2023-04-07 um 13 26 35" src="https://user-images.githubusercontent.com/932156/230601273-4fcb5892-28be-4f73-8f7f-6f57440e8623.png">

IMHO the notes have a good, distinctive color, so we'll leave them as-is for now.

**Comment by squidfunk** [comment]
Released as part of [9.1.6](https://github.com/squidfunk/mkdocs-material/releases/tag/9.1.6).
